Bulldozer.CSV/Maps/FinancialTransaction.cs

```diff
@@ -0,0 +1,295 @@
+            {
+                LoadGroupDict( rockContext );
+                groupMemberLookup = this.GroupDict.SelectMany( g => g.Value.Members )
+                                                        .Where( gm => !string.IsNullOrEmpty( gm.ForeignKey ) && gm.ForeignKey.StartsWith( ImportInstanceFKPrefix + "^" ) )
+                                                        .Select( a => new
+                                                        {
+                                                            GroupMember = a,
+                                                            a.ForeignKey
+                                                        } )
+                                                        .ToDictionary( k => k.ForeignKey, v => v.GroupMember );
+            }
+            ReportProgress( 0, string.Format( "Begin processing {0} FinancialTransaction Records...", this.FinancialTransactionCsvList.Count ) );
+
+
+            int giverAnonymousPersonAliasId = new PersonService( rockContext ).GetOrCreateAnonymousGiverPerson().Aliases.FirstOrDefault().Id;
+            var existingImportedTransactions = new FinancialTransactionService( rockContext ).Queryable().Where( a => a.ForeignKey != null && a.ForeignKey.StartsWith( this.ImportInstanceFKPrefix + "^" ) );
+            var existingImportedTransactionsHash = new HashSet<string>( existingImportedTransactions.Select( a => a.ForeignKey ).ToList() );
+            var personAliasIdLookup = ImportedPeopleKeys.ToDictionary( k => k.Key, v => v.Value.PersonAliasId );
+            var groupEntityTypeId = EntityTypeCache.Get( Rock.SystemGuid.EntityType.GROUP.AsGuid() ).Id;
+            var groupMemberEntityTypeId = EntityTypeCache.Get( Rock.SystemGuid.EntityType.GROUP_MEMBER.AsGuid() ).Id;
+
+            var accountIdLookup = ImportedAccounts.ToDictionary( k => k.Key, v => v.Value.Id );
+            var creditCardTypes = DefinedTypeCache.Get( new Guid( Rock.SystemGuid.DefinedType.FINANCIAL_CREDIT_CARD_TYPE ) ).DefinedValues;
+
+            // Slice data into chunks and process
+            var transactionsRemainingToProcess = this.FinancialTransactionCsvList.Count;
+            var workingTransactionCsvList = this.FinancialTransactionCsvList.ToList();
+            var completed = 0;
+
+            while ( transactionsRemainingToProcess > 0 )
+            {
+                if ( completed > 0 && completed % ( this.DefaultChunkSize * 10 ) < 1 )
+                {
+                    ReportProgress( 0, $"{completed} FinancialTransaction records processed." );
+                }
+                if ( completed % this.DefaultChunkSize < 1 )
+                {
+                    var csvChunk = workingTransactionCsvList.Take( Math.Min( this.DefaultChunkSize, workingTransactionCsvList.Count ) ).ToList();
+                    var errors = string.Empty;
```

Comment:
This looks out of place. Is it being used? If so, need to use it properly. Probably outside of while loop.